### DISCUSSION TITLE
VS Code Local Development Setup

### DISCUSSION THREAD

**Discussion by codewithunknown** [microsoft/vscode-discussions#1197] [Extension Development QnA] [answered]
Hello,

I'm a GSOC'24 contributor. I'm currently setting up VSCode on my local machine. I'm familiar with web development, and my tech stack includes React, Next.js, JavaScript, TypeScript, Node.js, and Spring. Firstly, I'm happy that I managed to set up VSCode for contributing, but are there any prerequisite languages or frameworks I should be familiar with for contributing? Also, for changes to reflect in the source code, is there any specific command? Additionally, as the codebase is huge, I'm having difficulty understanding the functionality. Which code is responsible, and how can I find specific code related to UI, functionality, or action?

can anyone please help me regarding this

ps : (I am using WSL)

![image](https://github.com/microsoft/vscode-discussions/assets/126901279/f86e3011-a404-4583-b64c-c9368bb2fcaf)

**Comment by codewithunknown** [comment]
Also ,

To start vs code I am using
```
./scripts/code.sh
```
this open a new vscode window as shown above is this the correct approach

**Comment by isidorn** [comment, marked as answer]
Please follow this https://github.com/microsoft/vscode/wiki/How-to-Contribute
That should help
